### PR TITLE
Fix formatting in OWC articles

### PR DESCRIPTION
**Pull request by respektive**
Noticed these errors in https://github.com/ppy/osu-wiki/pull/7097

I put the winning conditions under another heading, which is what the 2020/21 OWC articles do and removed the weird gaps in the stage instructions. Left the outdated articles untouched as they're outdated anyway.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/OWC/2014/en.md [hunk 1]
```diff
@@ -444,39 +444,30 @@ Sunday, 9 November 2014:
...
-    - In Semi-finals and Finals, you need to win 6 maps to win a match. (Best-of-11)
+| Stage | Match ID |
+| :-- | :-- |
+| Round of 16 | A, B, C, D, E, F, G, H |
+| Quarterfinals | I, J, K, L & R, S, T, U |
+| Semifinals | M, N & V, W, X, Y, Z, AA |
+| Finals week 1 | O & AB, AC, AD, AE |
+| Finals week 2 | P, Q |
+
+#### Winning conditions
```
Comment (Walavouchey) [inline wiki/Tournaments/OWC/2014/en.md:466]: you can interrupt a list with a table (without an indent), so you don't need another heading

**Review by Walavouchey — CHANGES_REQUESTED** [review]
as above

**Comment by Walavouchey** [comment]
see [the oSEAT5 article](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/oSEAT/5/en.md?plain=1#L870L889) for an example

File: wiki/Tournaments/OWC/2014/en.md (hunk 1 above)
Comment (respektive) [reply wiki/Tournaments/OWC/2014/en.md:466]: Oh, I didn't know that. Will adjust accordingly.

Comment (respektive) [reply wiki/Tournaments/OWC/2014/en.md]: reverted the heading

File: wiki/Tournaments/OWC/2014/en.md [hunk 2]
```diff
@@ -444,39 +444,29 @@ Sunday, 9 November 2014:
...
+| Round of 16 | A, B, C, D, E, F, G, H |
+| Quarterfinals | I, J, K, L & R, S, T, U |
+| Semifinals | M, N & V, W, X, Y, Z, AA |
+| Finals week 1 | O & AB, AC, AD, AE |
+| Finals week 2 | P, Q |
+
+12. **Win conditions:**
+   - In Group Stage, you need to win 4 maps to win a match. (Best-of-7)
+   - In the Round of 16 and the Quarter-finals, you need to win 5 maps to win a match. (Best-of-9)
+   - In Semi-finals and Finals, you need to win 6 maps to win a match. (Best-of-11)
```
Comment (Walavouchey) [inline wiki/Tournaments/OWC/2014/en.md:469]:
```suggestion
    - In Group Stage, you need to win 4 maps to win a match. (Best-of-7)
    - In the Round of 16 and the Quarter-finals, you need to win 5 maps to win a match. (Best-of-9)
    - In Semi-finals and Finals, you need to win 6 maps to win a match. (Best-of-11)
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**